core/src/main/java/org/apache/calcite/sql/SqlDialect.java

```diff
@@ -754,6 +754,7 @@ public boolean supportsGroupByLiteral() {
 
   public boolean supportsAggregateFunction(SqlKind kind) {
     switch (kind) {
+    case LITERAL_AGG:
```

Comment:
Do we need to add a test in `RelToSqlConverterTest`?

Reply:
The unit test testPushDownNotIn includes the plan and HSQLDB SQL(Rel To Sql). The changes to this PR have already been covered.